vision/api/Detect/Program.cs

```diff
@@ -251,6 +256,25 @@ private static object DetectText(Image image)
             return 0;
         }
 
+        private static object DetectTextWithLocation(Image image)
+        {
+            // [START vision_set_endpoint]
+
+            // Instantiate a client connected to the 'eu' location.
+            var client = new ImageAnnotatorClientBuilder()
+            {
+                Endpoint = new ServiceEndpoint("eu-vision.googleapis.com")
+            }.Build();
+            // [END vision_set_endpoint]
+

```

Comment:
Extraneous newline. Besides these 2 extra lines (I looked at the other samples in this file and they don't have empty newlines), all LGTM

Reply:
Done.

Reply:
Thank you!